server/profile.js

```diff
@@ -0,0 +1,42 @@
+const { Pool, Client } = require('pg')
+
+const pool = new Pool({
+  user: process.env.user,
+  host: process.env.host,
+  database: process.env.database,
+  password: process.env.password,
+  port: process.env.port,
+})
+
+
+module.exports.getProfile = async (event, context, callback) => {
+
+  let profileId = Number(event.queryStringParameters.id);
+
+  const selectProfile = "SELECT * FROM Cleaner_profile WHERE profile_id = $1";
+  const values = [profileId]
+
+  const result = await pool.query(selectProfile, values);
+
+  if(result.rowCount !=1){
```

Comment:
Is there any instance where the result can be > 1? In that case, maybe we should have `if (result.rowCount == 0)`. Also, as per the discussion around formatting from the retrospective, could we add a space between the parent and the brace?

Reply:
This call should only return 1 row. Otherwise there is an issue with the db. I am selecting based on the key, so only 1 answer should be returned.

Reply:
Cool, merci!